Project/DAQ_System/Program/Src/Platform/CAN/STM/F4/bxcan_stmf4.hpp

```diff
@@ -14,10 +14,70 @@
 #ifndef STM_F4_BXCAN_STMF4_H
 #define STM_F4_BXCAN_STMF4_H
 
+// Standard Libraries
+#include <vector>
+
+// ST HAL Dependencies
+#include "can.h"
+
+// DFR Custom Dependencies
+#include "../../Interfaces/ican.hpp"
+
+
 namespace platform {
 
-class BxCanStmF4 {
+class BxCanStmF4 : public ICan {
```

Comment:
It may just be me but what does this line do? Why does it have a ` : public ICan` after the class definition?

Reply:
This makes the class BxCanStmF4 inherit the member functions of ICan